lib/handlers/alias.get-v2.js

```diff
@@ -0,0 +1,145 @@
+import { validators } from "@eik/common";
+import originalUrl from "original-url";
+import HttpError from "http-errors";
+import abslog from "abslog";
+import Metrics from "@metrics/client";
+import Asset from "../classes/asset.js";
+
+import { decodeUriComponent, readJSON } from "../utils/utils.js";
+import {
+	createFilePathToAlias,
+	createFilePathToAsset,
+} from "../utils/path-builders-fs.js";
+import HttpOutgoing from "../classes/http-outgoing.js";
+import config from "../utils/defaults.js";
+
+/**
+ * @typedef {object} AliasGetOptions
+ * @property {boolean} [etag]
+ * @property {string} [cacheControl]
+ * @property {Array<[string, string]>} [organizations] List of key-value pairs [hostname, organization]
+ * @property {import("@eik/sink").default} [sink]
+ * @property {import("abslog").AbstractLoggerOptions} [logger]
+ */
+
+const AliasGet = class AliasGet {
+	/**
+	 * @param {AliasGetOptions} options
+	 */
+	constructor({ organizations, cacheControl, logger, sink, etag } = {}) {
+		this._organizations = organizations || config.organizations;
+		// 2 weeks stale while revalidate
+		this._cacheControl = cacheControl || "public, max-age=1200, stale-while-revalidate=1209600";
+		this._sink = sink;
+		this._etag = etag || config.etag;
+		this._log = abslog(logger);
+		this._metrics = new Metrics();
+		this._histogram = this._metrics.histogram({
+			name: "eik_core_alias_get_handler",
+			description:
+				"Histogram measuring time taken in @eik/core AliasGet handler method",
```

Comment:
This metric name collides with https://github.com/eik-lib/core/blob/047b61ddd917ee1e50689aed7de96b164e666d86/lib/handlers/alias.get.js#L32C11-L32C37

```suggestion
			name: "eik_core_alias_swr_get_handler",
			description:
				"Histogram measuring time taken in @eik/core stale-while-revalidate powered AliasGet handler method",
```